microsetta_interface/static/vue_survey_form_es_mx.js

```diff
@@ -0,0 +1,301 @@
+var vm = new Vue({
+    el: "#app",
+
+    components: {
+        "vue-form-generator": VueFormGenerator.component
+    },
+
+    data() {
+        return {
+            model: survey_model,
+            schema: survey_schema,
+            formOptions: {
+                validateAfterLoad: true,
+                validateAfterChanged: true
+            }
+        };
+    },
+
+    methods: {
+        prettyJSON: function(json) {
```

Comment:
A lot of the stuff here and below looks like it's replicated across multiple files. Can the common logic be centralized?